config/routes.rb

```diff
@@ -19,11 +19,15 @@
             get :update_amount, to: "order_details#update_amount"
           end
         end
+        collection do
+          get :search, to: "orders#search"
```

Comment:
bỏ to: "orders#search" thì còn chạy dc k e

Reply:
khác nhau giữa collection và member là gì?

Reply:
> bỏ to: "orders#search" thì còn chạy dc k e

bỏ được c =)))

Reply:
> khác nhau giữa collection và member là gì?

member tạo các route cho resource đơn lẻ (ex: resource/:id/xxx), còn collection là các route chorchính resource đấy (resources/xxx)